rllib/policy/policy.py

```diff
@@ -214,7 +214,8 @@ def postprocess_trajectory(self,
         Returns:
             SampleBatch: Postprocessed sample batch.
         """
-        return sample_batch
+        # Call our Exploration object's postprocess method.
+        return self.exploration.postprocess_trajectory(self, sample_batch)
```

Comment:
Instead of doing this in the policy, which would require all custom policies to implement this, why not call policy.exploration.postprocess_trajectory() unconditionally in sample_batch_builder.py:152 (after policy.postprocess_traj?)

Reply:
+1

Reply:
done